worker/migrations/migrations.go

```diff
@@ -237,6 +237,12 @@ func migrateAccountsToOrganization(domain string) error {
 	setting.ExtensionInstalled = true
 	settings.UpdateRevisionDate(inst, setting)
 	if err != nil {
+		if couchdb.IsConflictError(err) {
```

Comment:
:warning: It looks like the error doesn't come from saving the settings, but from before!